controllers/operandrequest_controller.go

```diff
@@ -69,18 +69,21 @@ func (r *OperandRequestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 	// Update labels for the request
 	if requestInstance.UpdateLabels() {
 		if err := r.Update(context.TODO(), requestInstance); err != nil {
+			klog.Errorf("Failed to update the labels for OperandRequest %s/%s : %v", req.NamespacedName.Namespace, req.NamespacedName.Name, err)
```

Comment:
I think you can use `req.NamespacedName.String()` to instead of `req.NamespacedName.Namespace, req.NamespacedName.Name`, like this:
```go
klog.Errorf("Failed to update the labels for OperandRequest %s : %v", req.NamespacedName.String(), err)
```

Reply:
Thanks, I update them in the following commit.